src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx

```diff
@@ -82,22 +82,34 @@ const ReleaseFootnotesSection = ({
     <>
       <h2>Footnotes</h2>
 
+      <p>
+        {!footnotesData.canUpdateRelease &&
+          !footnotesData.canUpdateReleaseFootnotes &&
+          'This release has been approved, and can no longer be updated.'}
+
+        {footnotesData.canUpdateRelease &&
+          !footnotesData.canUpdateReleaseFootnotes &&
+          'This release is an amendment to a live release and so it is not possible to change any footnotes.'}
+      </p>
+
       {footnotesData.footnoteMeta && (
         <>
-          {footnotesData.canUpdateRelease && (
-            <FootnoteForm
-              {...footnoteForm}
-              footnote={undefined}
-              onOpen={footnoteFormControls.create}
-              onCancel={footnoteFormControls.cancel}
-              onSubmit={footnoteFormControls.save}
-              isFirst={
-                footnotesData.footnotes && footnotesData.footnotes.length === 0
-              }
-              footnoteMeta={footnotesData.footnoteMeta}
-              footnoteMetaGetters={footnotesData.footnoteMetaGetters}
-            />
-          )}
+          {footnotesData.canUpdateRelease &&
+            footnotesData.canUpdateReleaseFootnotes && (
+              <FootnoteForm
+                {...footnoteForm}
+                footnote={undefined}
+                onOpen={footnoteFormControls.create}
+                onCancel={footnoteFormControls.cancel}
+                onSubmit={footnoteFormControls.save}
+                isFirst={
+                  footnotesData.footnotes &&
+                  footnotesData.footnotes.length === 0
```

Comment:
Could make this statement simpler with `!foonotesData.footnotes?.length`

Reply:
yep - done - thanks